pushmsixscript/docker.d/init_worker.sh

```diff
@@ -15,29 +15,56 @@ export LOGIN_URL=https://login.microsoftonline.com
 export TOKEN_RESOURCE=https://manage.devcenter.microsoft.com
 export STORE_URL=https://manage.devcenter.microsoft.com/v1.0/my/applications/
 export MOCK_APPLICATION_ID=MOCK-APP-ID
-export BETA_APPLICATION_ID=9NZW26FRNDLN
-export RELEASE_APPLICATION_ID=9NZVDKPMR9RD
 export RELEASE_ROLLOUT_PERCENTAGE=25.0
 
-case $ENV in
-  dev|fake-prod)
+case $COT_PRODUCT in
+  firefox)
+    export BETA_APPLICATION_ID=9NZW26FRNDLN
+    export RELEASE_APPLICATION_ID=9NZVDKPMR9RD
     ;;
-  prod)
-    test_var_set 'TENANT_ID'
-    test_var_set 'CLIENT_ID'
-    test_var_set 'CLIENT_SECRET'
-    export TENANT_ID=`echo $TENANT_ID | base64 -d`
-    export CLIENT_ID=`echo $CLIENT_ID | base64 -d`
-    export CLIENT_SECRET=`echo $CLIENT_SECRET | base64 -d`
+  thunderbird)
+    export BETA_APPLICATION_ID=9PFQWVMSS45P
+    export RELEASE_APPLICATION_ID=9PM5VM1S3VMQ
     ;;
   *)
-    exit 1
+    # Product validation below will handle this case
     ;;
 esac
 
-case $COT_PRODUCT in
-  firefox) ;;
+
+case $ENV in
+  dev|fake-prod)
+      case $COT_PRODUCT in
+        # dev and fake-prod do not have variables set
+        firefox|thunderbird)
+          ;;
+        *)
+          echo "FAIL: Unsupported product in dev environment: ${COT_PRODUCT}" >&2
+          exit 1
+          ;;
+      esac
+      ;;
+  prod)
+    case $COT_PRODUCT in
+      firefox|thunderbird)
+        # For firefox prod environment
+        test_var_set 'TENANT_ID'
+        test_var_set 'CLIENT_ID'
+        test_var_set 'CLIENT_SECRET'
+        export TENANT_ID=$(echo "$TENANT_ID" | base64 -d)
+        export CLIENT_ID=$(echo "$CLIENT_ID" | base64 -d)
+        export CLIENT_SECRET=$(echo "$CLIENT_SECRET" | base64 -d)
```

Comment:
Do we have values for these already, or are we waiting on someone from Thunderbird or elsewhere to provide them?

Reply:
Yes, already added them to SOPS.